tfjs-converter/python/tensorflowjs/BUILD.bazel

```diff
@@ -110,6 +110,12 @@ py_library(
     deps = [requirement("packaging")],
 )
 
+py_library(
```

Comment:
Note to reviewer: debugpy is already in google3, so this shouldn't break copybara.